packages/translate/src/index.js

```diff
@@ -268,6 +269,7 @@ Translate.prototype.getLanguages = function(target, callback) {
  *     target ISO 639-1 language code to translate the source input to. (e.g.
  *     `en` for English). If an object, you may also specify the source
  *     language.
+ * @param {string} options.format - Set the text's format as `html` or `text`.
```

Comment:
`options` itself is marked as optional. I elaborated on what happens if a user doesn't provide a format.